Modules/Axis/Tests/AxisTests/String/StringTests.swift

```diff
@@ -75,13 +75,13 @@ public class StringTests : XCTestCase {
         XCTAssertThrowsError(try String(percentEncoded: "%00%FF%FF%00"))
     }
 
-    func testURLQueryPercentEncoding() {
-        XCTAssertEqual("abc".percentEncoded(allowing: .uriQueryAllowed), "abc")
-        XCTAssertEqual("joão".percentEncoded(allowing: .uriQueryAllowed), "jo%C3%A3o")
-        XCTAssertEqual("💩".percentEncoded(allowing: .uriQueryAllowed), "%F0%9F%92%A9")
-        XCTAssertEqual("foo bar".percentEncoded(allowing: .uriQueryAllowed), "foo%20bar")
-        XCTAssertEqual("foo\nbar".percentEncoded(allowing: .uriQueryAllowed), "foo%0Abar")
-    }
+//    func testURLQueryPercentEncoding() {
```

Comment:
Reason why I commented this test method out is because it tests this (also commented out):

````swift
//    public func percentEncoded(allowing allowed: Characters) -> String {
//        var string = ""
//        let allowed = allowed.utf8()
//
//        for codeUnit in self.utf8 {
//            if allowed.contains(codeUnit) {
//                string.append(String(UnicodeScalar(codeUnit)))
//            } else {
//                string.append("%")
//                string.append(codeUnit.hexadecimal())
//            }
//        }
//
//        return string
//    }
````

.. because it's using `Charater`s thing which I am trying to get away from in this PR.

Actually, this reminds me, the whole type `Characters` is redundant now.  Should I maybe just slash them all out (delete, that is)?